Lib/_pydecimal.py

```diff
@@ -886,35 +886,31 @@ def __lt__(self, other, context=None):
         self, other = _convert_for_comparison(self, other)
         if other is NotImplemented:
             return other
-        ans = self._compare_check_nans(other, context)
-        if ans:
+        if self._compare_check_nans(other, context):
```

Comment:
This doesn't even use the assignment expression

Reply:
@graingert 
Correct. It seems to be a minor code cleanup. `ans` is not really used except in the `if`, so can be done away with.